src/github.com/matrix-org/go-neb/services/rssbot/rssbot.go

```diff
@@ -214,6 +220,7 @@ func itemToHTML(feed *gofeed.Feed, item gofeed.Item) matrix.HTMLMessage {
 }
 
 func init() {
+	cachingClient = httpcache.NewMemoryCacheTransport().Client()
```

Comment:
No, though it looks like we can plug-and-play an [LruCache](https://github.com/die-net/lrucache).